java/plugins/jdi-elements-plugin/src/test/java/ru/sbtqa/tag/pagefactory/CucumberTest.java

```diff
@@ -0,0 +1,12 @@
+package ru.sbtqa.tag.pagefactory;
+
+import cucumber.api.CucumberOptions;
+import org.junit.runner.RunWith;
+import ru.sbtqa.tag.cucumber.TagCucumber;
+
+@RunWith(TagCucumber.class)
```

Comment:
Have we already avoid of TagCucumber? (In PF 1.8 we already did it I guess)

Reply:
nope, we are not yet ready for this because of https://github.com/sbtqa/page-factory-2/blob/4cbf45e65e3f0731e266f505e7e8ad65d2f33761/java/core/page-factory-utils/src/main/java/ru/sbtqa/tag/pagefactory/util/PageFactoryUtils.java#L103